src/System.Net.Sockets/src/System/Net/Sockets/NetworkStream.cs

```diff
@@ -614,6 +614,9 @@ internal bool Connected
         // Returns:
         // 
         //     An IASyncResult, representing the read.
+#if !netcore50
+        override
+#endif
```

Comment:
@davidsh removing the live netcore50 build doesn't break UWP. It just implies that we are going to pull the netcore50 version of the library from the last stable shipped package and reship that in the current package. It just allows us to eliminate a bunch of the extra live build configurations.